projects/plugins/jetpack/extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -31,6 +31,15 @@ function register_block() {
 			BLOCK_NAME,
 			array(
 				'render_callback' => __NAMESPACE__ . '\load_assets',
+				'supports'        => array(
+					'align'   => array( 'wide', 'full' ),
+					'color'   => array(
+						'gradients' => true,
```

Comment:
> Missing link here.

I have experimented with adding the Link color as well, but it would apply the color of the links in the text below the form only:

![Markup on 2022-02-09 at 09:43:19](https://user-images.githubusercontent.com/25105483/153158328-ae19bad7-f6a5-4e86-b0a8-0ba4180a7400.png)

A bit of a confusion I felt, plus expecting that not many users would use that anyways made me omit the Link color setting.

Reply:
Oh the initial confusion was around `link` property was set in JS, but not in PHP. And I see what you mean by users not understanding why the link panel is there when there are no links initially on the mailchimp block. 

Decision is to omit link.